RecoLocalTracker/SiStripClusterizer/plugins/SiStripApprox2ApproxClusters.cc

```diff
@@ -0,0 +1,90 @@
+
+
+#include "FWCore/Framework/interface/MakerMacros.h"
+#include "FWCore/Framework/interface/Frameworkfwd.h"
+#include "FWCore/Framework/interface/stream/EDProducer.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/Utilities/interface/InputTag.h"
+#include "FWCore/ParameterSet/interface/ConfigurationDescriptions.h"
+#include "FWCore/ParameterSet/interface/ParameterSetDescription.h"
+#include "DataFormats/SiStripCluster/interface/SiStripApproximateCluster.h"
+#include "DataFormats/SiStripCluster/interface/SiStripCluster.h"
+#include "DataFormats/Common/interface/DetSetVectorNew.h"
+#include "DataFormats/Common/interface/DetSetVector.h"
+
+#include <vector>
+#include <memory>
+
+class SiStripApprox2ApproxClusters : public edm::stream::EDProducer<> {
+public:
+  explicit SiStripApprox2ApproxClusters(const edm::ParameterSet& conf);
+  void produce(edm::Event&, const edm::EventSetup&) override;
+
+  static void fillDescriptions(edm::ConfigurationDescriptions& descriptions);
+
+private:
+  edm::InputTag inputApproxClusters;
+  std::string approxVersion;
+  edm::EDGetTokenT<edmNew::DetSetVector<SiStripApproximateCluster>> clusterToken;
+};
+
+SiStripApprox2ApproxClusters::SiStripApprox2ApproxClusters(const edm::ParameterSet& conf) {
+  inputApproxClusters = conf.getParameter<edm::InputTag>("inputApproxClusters");
+  approxVersion = conf.getParameter<std::string>("approxVersion");
+
+  clusterToken = consumes<edmNew::DetSetVector<SiStripApproximateCluster>>(inputApproxClusters);
+  produces<edmNew::DetSetVector<SiStripApproximateCluster>>();
+}
+
+void SiStripApprox2ApproxClusters::produce(edm::Event& e, edm::EventSetup const&) {
```

Comment:
very short variable names are not ideal for something that has a longer scope (readability)
```suggestion
void SiStripApprox2ApproxClusters::produce(edm::Event& event, edm::EventSetup const&) {
```

Reply:
done